client/python/gradio_client/utils.py

```diff
@@ -413,6 +421,75 @@ async def stream_sse(
         raise
 
 
+async def stream_sse_v1(
+    client: httpx.AsyncClient,
+    data: dict,
+    hash_data: dict,
+    helper: Communicator,
+    sse_url: str,
+    sse_data_url: str,
+    cookies: dict[str, str] | None = None,
+) -> dict[str, Any]:
+    try:
+        req = await client.post(
+            sse_data_url,
+            json={**data, **hash_data},
+            cookies=cookies,
+        )
+        req.raise_for_status()
+        resp = req.json()
+        event_id = resp["event_id"]
+
+        async with client.stream(
+            "GET", sse_url, params={"event_id": event_id}, cookies=cookies
```

Comment:
Needs to include `headers` as done here: https://github.com/gradio-app/gradio/pull/6602